docs/how-to/how-to-enable-storage-in-confidential-pods.md

```diff
@@ -228,32 +231,22 @@ experience this error.
 ## Installing the CoCo Azure Files driver
```

Comment:
We should:

* Remove the contents of this section and link to the official AKS Azure Files docs.
* Add the known limitations to the official AKS docs.

Reply:
Errata: Let's keep the known limitations in this doc for now. First point still stands.